src/utils/utf8.c

```diff
@@ -25,15 +26,24 @@
 #include <unicode/utypes.h>
 #include <unicode/ucasemap.h>
 #include <unicode/unorm2.h>
+#include <unicode/unistr.h>
+#include <unicode/utrans.h>
 
 #include "strbuf.h"
 #include "utf8.h"
 #include "utils.h"
 
+/*
+* UCI Rule for transliteration.
+* see more: https://unicode-org.github.io/icu/userguide/transforms/general/
+*/
+const char * RULE = "NFKD; [:M:] Remove; Lower; Latin-ASCII; NFKC";
```

Comment:
- "RULE" is too generic.  How about `NFKD_RULE`?
- This should be `static const char`. You can also use the bracket notation, i.e.: `static const char NFKD_RUKE[] = ...`. This way you can use `sizeof(NFKD_RULE) - 1` instead of O(n) strlen.